### DISCUSSION TITLE
This branch has conflicts that must be resolved | Git Bash

### DISCUSSION THREAD

**Discussion by AlisonStrand** [community/community#146093] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I´m a new user and I have a question about the conflicts that should be resolved by using Git bash, but when I use it i does not show any problem at all. 

(On bash)
$ git status
On branch Assets
Your branch is up to date with 'origin/Assets'.

nothing to commit, working tree clean

So far this is what is showing me, so does anyone know how to fix this?

Thank you

**Comment by spenserblack** [comment]
So you're getting the conflict message on the GitHub site, right? What this message means is that GitHub can't automatically merge, because a coder needs to decide how the merge should happen. Here's a simple way to recreate the issue in a test repository:
1. Create a repository with a single file called `test.txt` with the contents
   ```
   test
   ```
2. Commit that file
3. Create a new branch called `changes`, make this change, commit and push:
   ```diff
   -test
   +test.
   ```
4. Switch back to your `main` branch, make this change, commit and push:
   ```diff
   -test
   +test!
   ```
5. Now, if you try to merge the branches, you'll get a merge conflict. This is because Git sees 2 changes in the same location, and needs you to decide how it should be handled (do you pick one or the other, or somehow combine them).

Additionally, binary (non-text) will pretty much always have merge conflicts if you change the same file in 2 branches. Binary files include image files, video files, and some data formats.

There should be instructions on GitHub with the commands to use to resolve merge conflicts, but here's how you can do it:

```shell
# on your branch Assets
git pull origin main
# you should now be prompted to resolve merge conflicts
# after resolving all merge conflicts:
git add CONFLICTING_FILES
git commit -m "Resolve conflicts"
git push
```

If your conflicts include binary files, which I'm thinking is possible considering the branch is called `Assets`, it can get bit more complicated. Let me know if you need additional help.

**Reply by AlisonStrand** [reply]
Thank you so much for your help!
